pkg/controller/network_policy.go

```
@@ -577,6 +671,25 @@ func (c *Controller) podMatchNetworkPolicies(pod *corev1.Pod) []string {
 	return match
 }
 
+func (c *Controller) svcMatchNetworkPolicies(svc *corev1.Service) []string {
+	//  find all match pod
+	pods, _ := c.podsLister.Pods(svc.Namespace).List(labels.Everything())
+
+	// find all match netpol
+	nps, _ := c.npsLister.NetworkPolicies(corev1.NamespaceAll).List(labels.Everything())
+	match := []string{}
+	for _, pod := range pods {
+		podNs, _ := c.namespacesLister.Get(pod.Namespace)
```

We shouldn't ignore the errors.

---

ok